ui/jstests/test-learning-resource.jsx

```diff
@@ -228,6 +229,7 @@ define(['QUnit', 'jquery', 'reactaddons', 'lodash', 'learning_resources',
 
           var saveButton = $node.find("button")[0];
           React.addons.TestUtils.Simulate.click(saveButton);
+          assert.equal(component.state.message, undefined);
```

Comment:
Whenever `setState` is called, even if triggered by an event, you must always assert the state change within the `forceUpdate` callback to ensure the state change was made.